test/Benchmarks/src/Main.enso

```diff
@@ -1,13 +1,85 @@
+from Standard.Base import all
+from Standard.Test import Bench
+import project.Vector.Array_Proxy_Bench
 import project.Vector.Distinct
 import project.Vector.Operations
+import project.Vector.Sort
+import project.Statistics.Count_Min_Max
+import project.Table.Aggregate
+import project.Table.Cross_Tab
+import project.Table.Sorting
+import project.Text.Build
+import project.Text.Compare
+import project.Text.Contains
+import project.Text.Pretty
+import project.Text.Reverse
+import project.Time.Work_Days
+import project.Collections
+import project.Column_Numeric
+import project.Equality
+import project.Json_Bench
+import project.Natural_Order_Sort
+import project.Number_Parse
 import project.Numeric
+import project.Sum
+from Standard.Base.Runtime import Debug
 
+all_benchmarks : Vector Bench.All
 all_benchmarks =
-    vec_ops = Operations.all
-    vec_distinct = Distinct.collect_benches
-    numeric = Numeric.bench
-    [vec_ops, vec_distinct, numeric]
+    builder = Vector.new_builder
+
+    # Vector
+    builder.append Array_Proxy_Bench.collect_benches
```

Comment:
Shouldn't all the `collect_benches` method accept a `Bench.build`er? Then this would be:
```
Bench.build builder ->
  Array_Proxy_Bench.collect_benches builder
  Distinct.callect_benches builder
  ...
```
Then we wouldn't need to iterate `Vector`, but rather:
```
main =
    all_benchmarks . run_main
```